internal/app/secretless/handlers/mysql/backend.go

```diff
@@ -126,6 +116,19 @@ func (h *Handler) ConnectToBackend() (err error) {
 		return
 	}
 
+	// Remove Client SSL Capability from Server Handshake Packet
+	serverHandshake.ServerCapabilities &^= protocol.ClientSSL
```

Comment:
In general, clients (in the code sense) should never be manipulating object member data themselves.  Instead, we should make a method to do it.  So in this case, we could write:
```go
serverHandshake.removedClientSSL()
```
and the bit masking logic would then be hidden from client code.